plugin/trino-opa/src/main/java/io/trino/plugin/opa/schema/OpaColumnMaskQueryResult.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.opa.schema;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public record OpaColumnMaskQueryResult(@JsonProperty("decision_id") String decisionId, @NotNull Optional<OpaViewExpression> result)
```

Comment:
My bad , I thought this was useful for denoting that built instances would never have a null value for `result`.

For cases like `OpaBatchQueryResult`: https://github.com/trinodb/trino/blob/74e3ffa65e18d2db7938d5e17b6036b8dac42efc/plugin/trino-opa/src/main/java/io/trino/plugin/opa/schema/OpaBatchQueryResult.java#L24-L30

I wanted to make it clear that the `result` field would never be null once the object is constructed, but it _may_ be null upon calling the constructor (in which case it'll be defaulted to an empty list).

Since `NotNull` targets the field in every context, should it just be removed everywhere? I'll remove them for now, but please lmk if I'm misunderstanding!